container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -32,6 +32,9 @@ const (
 
 var noWait *port_spec.Wait = nil
 
+// TODO add support for passing toleration to Engine
```

Comment:
is this TODO for the next iterations?

Reply:
Yes! https://github.com/kurtosis-tech/kurtosis/issues/2073